packages/puppeteer-core/src/common/Input.ts

```diff
@@ -682,4 +682,42 @@ export class Touchscreen {
       modifiers: this.#keyboard._modifiers,
     });
   }
+
+  /**
+   * Dispatches a `touchstart` event.
+   * @param x - Horizontal position of the tap.
+   * @param y - Vertical position of the tap.
+   */
+  async touchStart(x: number, y: number): Promise<void> {
```

Comment:
let's use touchStart and touchEnd in tap above

Reply:
ok